lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -1,7 +1,4 @@
-//////////////////////////////////
-// BEGIN sdk-common mirror imports
-// These are structs defined in sdk-common, which we want to make available in this project's UDL bindings
-
```

Comment:
Could these lines be preserved? They would make life easier in the future when dealing with mirror structs.

Reply:
These comments where breaking uniffi when followed by a docstring e.g.
```
// BEGIN sdk-common mirror imports
// These are structs defined in sdk-common, which we want to make available in this project's UDL bindings

/// Wrapper for a BOLT11 LN invoice
dictionary LNInvoice {
```
I'm not sure why, I can remove the LNInvoice docstring if its important. 

Is this an improvement if I comment the end of the block? [6caddd9](https://github.com/breez/breez-sdk-liquid/pull/512/commits/6caddd980a312664dc8f96d5877895c4a0cc2636)

Reply:
> These comments where breaking uniffi when followed by a docstring e.g.

Would it work with a comment block like in 6caddd980a312664dc8f96d5877895c4a0cc2636 ?

> Is this an improvement if I comment the end of the block? 6caddd980a312664dc8f96d5877895c4a0cc2636

Yes, thanks.